jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -529,6 +529,11 @@ public Module addEnableNativeAccess(Module mod) {
 	public boolean addEnableNativeAccess(ModuleLayer moduleLayer, String moduleName) {
 		return moduleLayer.addEnableNativeAccess(moduleName);
 	}
+
+	@Override
+	public boolean allowSecurityManager() {
```

Comment:
Should inclusion really be controlled by !INLINE-TYPES ?

Reply:
Alternatively `@Override` can be removed, but another PR is still required to add it back after [valhalla extension](https://github.com/ibmruntimes/openj9-openjdk-jdk.valuetypes) is merged with the latest [openjdk head-stream](https://github.com/ibmruntimes/openj9-openjdk-jdk) contents, or just no `@Override`?

FYI @keithc-ca

Reply:
Do we have an estimate of when the Valhalla code will catch up?
If the update to [ModuleLayer](https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/76350f2225a42234f5f0b263badebdc725839a29) is not present, this won't compile (with or without `@Override`).

Reply:
> Do we have an estimate of when the Valhalla code will catch up?

[valhalla extension](https://github.com/ibmruntimes/openj9-openjdk-jdk.valuetypes) is based on [openjdk feature stream](https://github.com/openjdk/valhalla) which at [jdk-22+25](https://github.com/openjdk/valhalla/commit/bfd689160c67449ea601dd7bfa9c5b1114b25558), there are large number of file changes within `java.base/share/classes/jdk/internal/classfile` still pending.

Reply:
If valhalla builds use `JAVA_SPEC_VERSION=22` (as it would seem they should), then this code would not be included.

Reply:
[openjdk feature stream](https://github.com/openjdk/valhalla) aims to merge with [openjdk head](https://github.com/openjdk/jdk), so does [valhalla extension](https://github.com/ibmruntimes/openj9-openjdk-jdk.valuetypes).

Reply:
Yes, but on the default branch (lworld), it's still at Java 22; see [version-numbers.conf](https://github.com/openjdk/valhalla/blob/lworld/make/conf/version-numbers.conf#L29).

Reply:
Same is true in our repo; see [version-numbers.conf](https://github.com/ibmruntimes/openj9-openjdk-jdk.valuetypes/blob/openj9/make/conf/version-numbers.conf#L32). The condition `!INLINE-TYPES` doesn't help.

Reply:
I'm not sure of the next step for this. Are we just going to merge it as is (which is fine with me) to fix jdknext builds?

Reply:
My preference is to remove the `!INLINE-TYPES` part of the preprocessor condition. It isn't the appropriate long-term answer and based on my earlier comments isn't necessary now. Otherwise, this looks good.

Reply:
Trying a valhalla build to verify w/o `!INLINE-TYPES`.

Reply:
Removed `!INLINE-TYPES` since `valhalla` is still in JDK 22 level.